src/js/app.spec.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @jest-environment jsdom
+ */
+import main from './app.js';
+import { addMessageToChatHistory } from './chatWindow.js';
+import { onEnterPress, onFormSubmit } from './eventHandlers.js';
+
+jest.mock('./eventHandlers.js');
+jest.mock('./chatWindow.js');
+
+describe('app.js', () => {
+  it('Runs onEnterPress with addMessageToChatHistory as a parameter', () => {
+    main();
+    expect(onEnterPress).toHaveBeenCalledWith(addMessageToChatHistory);
+  });
+  it('Runs onFormSubmit with addMessageToChatHistory as a parameter', () => {
+    main();
+    expect(onFormSubmit).toHaveBeenCalledWith(addMessageToChatHistory);
+  });
```

Comment:
This is just a code-style preference thing, but I'd add an empty line between the tests to break them up visually. Literally just nitpicking because everything else is so solid 😂

Reply:
I agree that it makes it easier to read, added this as well!